src/main/java/dinkplugin/notifiers/ChatNotifier.java

```diff
@@ -48,14 +50,15 @@ public void onConfig(String key, String value) {
         }
     }
 
-    public void onMessage(ChatMessageType messageType, String message) {
+    public void onMessage(@NotNull ChatMessageType messageType, @Nullable String source, @NotNull String message) {
         ChatNotificationType type = ChatNotificationType.MAPPINGS.get(messageType);
         if (type != null && config.chatMessageTypes().contains(type) && isEnabled() && hasMatch(message)) {
-            this.handleNotify(messageType, message);
+            String cleanSource = source != null ? source.replace('\u00A0', ' ') : null;
+            this.handleNotify(messageType, cleanSource, message);
         }
     }
 
-    private void handleNotify(ChatMessageType type, String message) {
+    private void handleNotify(ChatMessageType type, String source, String message) {
         String playerName = Utils.getPlayerName(client);
         Template template = Template.builder()
```

Comment:
if we want `%SOURCE%` added as template var, what should be the default value when source is null?
(if we don't want this replacement, ignore this)

Reply:
I'd say no replacement in this PR since there's no perfect answer to this question